### ISSUE TITLE
torch.max() document bug

### ISSUE THREAD

**Issue by Hao-Kailong** [pytorch/pytorch#49845]
## 📚 Documentation

in  document torch.max()
https://pytorch.org/docs/stable/generated/torch.max.html?highlight=torch%20max#torch.max

> If there are multiple maximal values in a reduced row then the indices of the first maximal value are returned.

while when I tried those codes below, it is in contrast with the document:
~~~python
x = torch.tensor([1,2,3,7,5,4,7,1,3], dtype=torch.int)
_, tmp = torch.max(x, dim=0)
print(tmp)
~~~

I got:
~~~bash
tensor(6)
~~~

it seems that the indice of the last maximal value is returned. instead of the first one.

**so I suggest to change the document to this:**
> If there are multiple maximal values in a reduced row then the indices of the last maximal value are returned.

**Comment by C3Suryansu** [comment]
I tried the same code as yours, but, I got the output as 
```
tensor(3)
``` 
which is in accordance with the pytorch documentation.
![image](https://user-images.githubusercontent.com/39475196/103124691-8ab05d80-46ae-11eb-9407-a7b316327dfb.png)

**Comment by Hao-Kailong** [comment]
My configuration is below:
~~~json
python == 3.7.7
torch == 1.5.0
windows 10
~~~
And I'm using cpu version pytorch on Windows, without CUDA.

 ---

BUT I then tried on Linux, configuration is:
~~~json
python == 3.6.8
torch == 1.7.1+cu101
Linux version 4.15.0-66-generic (buildd@lgw01-amd64-044) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) 
~~~
I got the same result as yours. I got:
~~~bash
tensor(3)
~~~

**Maybe it's the cause of windows/pytorch version or something else?**

**Comment by C3Suryansu** [comment]
I guess so, the issue is being caused by pytorch/python version on windows. Maybe someone else can give better insight on this.

**Comment by ngimel** [comment]
This behavior is documented for pytorch version 1.7+ and happens in pytorch version 1.7+. On 1.5, the behavior and documentation was different
